kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Waitable.java

```diff
@@ -20,7 +20,19 @@
 
 public interface Waitable<T, P> {
 
-    T waitUntilReady(long amount, TimeUnit timeUnit) throws InterruptedException;
+  long DEFAULT_INITIAL_BACKOFF_MILLIS = 5;
+  double DEFAULT_BACKOFF_MULTIPLIER = 2;
```

Comment:
nit: Could you initialize values with correct primitive type to remove ambiguity
```suggestion
  long DEFAULT_INITIAL_BACKOFF_MILLIS = 5L;
  double DEFAULT_BACKOFF_MULTIPLIER = 2d;
```